css/style.css

```diff
@@ -587,7 +571,7 @@ margin-left:auto;
 
 
 
-@media screen and (max-width: 1279px) and (min-width:768px){
+@media screen and (max-width: 1024px) and (min-width:600px){

```

Comment:
- Maybe have 2 separate breakpoints for tablets (one for landscape, one for portrait)